.eslintrc

```diff
@@ -17,29 +17,26 @@
 
   "rules": {
     "prettier/prettier": "error",
-    "max-len": ["warn", { "code": 140, "ignoreComments": true, "ignoreUrls": true }],
     "no-undef": "warn",
     "no-unused-vars": "warn",
     "prefer-const": "warn",
     "camelcase": "off",
     "eol-last": "warn",
     "no-multiple-empty-lines": "warn",
-    "comma-dangle": "warn",
     "quotes": ["error", "single", { "avoidEscape": true, "allowTemplateLiterals": true }],
     "jsx-quotes": ["error", "prefer-single"],
     "no-useless-escape": "off",
-    "linebreak-style": ["error", "unix"]
+    "linebreak-style": ["error", "unix"],
+    "no-restricted-modules": ["error", {
+        "name": "@aragon/contract-helpers-test/src/asserts",
+        "message": "Please use helpers/assert instead."
```

Comment:
:+1: 
though, maybe it's better to make the fix more precise (specify path from the project's root)